### PR TITLE
INREL-4375 js cleanup

### PR DESCRIPTION
**Pull request by Nicozilla**
## [INREL-4375](https://jira.burda.com/browse/INREL-4375) 

- Move blocking JS files in <head>
- Remove unnecessary CSS files
- Remove unnecessary JS files
- Change JS stickiness to css:sticky
- Remove bootstrap dependencies 

## How to test:
- jQuery is not in <head>
- Module CSS are removed [Ticket]https://jira.burda.com/browse/INREL-4456)
- Bootstrap JS files are removed (modal.js / transition.js)
- Sticky elements (ads and socials) are sticky. (Socials on InStyle are not sticky)

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
- [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: js/infinite/models/device-model.js [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
       isActive: false,
       uid: -1,
-      uid_cookie_name: "tc_ptid",
-      basehost: window.location.hostname.split(".")[0],
+      uid_cookie_name: 'tc_ptid',
+      basehost: window.location.hostname.split('.')[0],
       cookieReferrerName: '_referrer',
       isGoogleBot: navigator.userAgent.toLowerCase().indexOf('googlebot') > 0,
-      useWhatsapp: ((navigator.userAgent.match(/Android|iPhone/i) && !navigator.userAgent.match(/iPod|iPad/i)) ? true : false)
+      useWhatsapp: (!!((navigator.userAgent.match(/Android|iPhone/i) && !navigator.userAgent.match(/iPod|iPad/i)))),
```
Comment (danschalow) [inline js/infinite/models/device-model.js]: Do we need all those brackets?

```javascript
useWhatsapp: !!navigator.userAgent.match(/Android|iPhone/i) && !navigator.userAgent.match(/iPod|iPad/i)
```

File: sass/mixins/_mixins.contents.scss [hunk 2]
```diff
@@ -168,6 +168,27 @@
     margin-left: $feed-ad-column-margin;
   }
 
+  .item-sticky {
+    $stickyOffset: $menu--main-navigation-min-height;
+    position: -webkit-sticky;
+    position: sticky;
+    top: 10px;
```
Comment (danschalow) [inline sass/mixins/_mixins.contents.scss]: Magic number?

File: sass/mixins/_mixins.socials.scss [hunk 3]
```diff
@@ -1,3 +1,23 @@
+@mixin socials-vertical-bar() {
+  $stickyOffset: $menu--main-navigation-min-height;
+  position: sticky;
+  top: 10px;
```
Comment (danschalow) [inline sass/mixins/_mixins.socials.scss]: Magic number

File: sass/variables/_variables.admin.scss [hunk 4]
```diff
@@ -0,0 +1,2 @@
+$toolbar-administration__height--closed: 39px;
+$toolbar-administration__height--open: 79px;
```
Comment (danschalow) [inline sass/variables/_variables.admin.scss:2]: Magic number

**Review by danschalow — CHANGES_REQUESTED** [review]
Few nit picks

File: js/infinite/models/device-model.js (hunk 1 above)
Comment (Nicozilla) [reply js/infinite/models/device-model.js]: changed

File: sass/mixins/_mixins.contents.scss (hunk 2 above)
Comment (Nicozilla) [reply sass/mixins/_mixins.contents.scss]: changed

File: sass/mixins/_mixins.socials.scss (hunk 3 above)
Comment (Nicozilla) [reply sass/mixins/_mixins.socials.scss]: changed

File: sass/variables/_variables.admin.scss (hunk 4 above)
Comment (Nicozilla) [reply sass/variables/_variables.admin.scss:2]: This is only for the admin menu. Got no other possibilities.

Comment (danschalow) [reply sass/variables/_variables.admin.scss:2]: 💋

**Review by danschalow — APPROVED** [review]
LGTM :shipit:

**Pull request merged**